src/modules/site-v2/base/views/tools/phenotype_database.py

```diff
@@ -92,7 +92,7 @@ def download_csv():
   def generate():
     yield file_format['sep'].join(columns) + '\n'
     try:
-      for row in traits.yield_per(1000):
+      for row in traits.yield_per(100):
```

Comment:
avoid magic numbers -- consider creating a variable that describes the meaning of `100`, e.g.: `TRAITS_PER_CHUNK=100`, etc.